src/Compilers/CSharp/Test/Semantic/Semantics/ConstantTests.cs

```diff
@@ -3645,6 +3655,11 @@ public A(string name)
 [A($""ITEM"")]
 class C
 {
+    class Namae
```

Comment:
Please add a const field to this test as well.

Reply:
Good call, see the new commit.